components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4738,6 +4738,9 @@ where
             "region" => ?region,
         );
 
+        // Update the flashback flag according to the snapshot region meta.
+        self.fsm.peer.is_in_flashback = region.is_in_flashback;
```

Comment:
Remove `is_in_flashback` field and use `region.is_in_flashback` as only source of truth. `set_region` would update it after applying snapshot. And `on_set_flashback_state` should update region as well.

Reply:
Nice idea. Already addressed. PTAL.